frontend/src/routes/Governance/discovered/ByCluster/common.tsx

```diff
@@ -0,0 +1,201 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { useMemo } from 'react'
+import { ViolationsCard, ViolationSummary } from '../../overview/PolicyViolationSummary'
+import { DiscoverdPolicyItem, DiscoverdPolicyTableItem, ISourceType } from '../useFetchPolicies'
+import { compareStrings, IAcmTableColumn } from '../../../../ui-components'
+import { TFunction } from 'react-i18next'
+import { getPolicySource, getStringLabelValue } from '../../common/util'
+import { generatePath, Link } from 'react-router-dom-v5-compat'
+import { NavigationPath } from '../../../../NavigationPath'
+import { Channel, HelmRelease, Subscription } from '../../../../resources'
+import { CheckCircleIcon, ExclamationCircleIcon, ExclamationTriangleIcon } from '@patternfly/react-icons'
+import { useTranslation } from '../../../../lib/acm-i18next'
+
+export const policyViolationSummary = (discoveredPolicyItems: DiscoverdPolicyItem[]): ViolationSummary => {
+  let compliant = 0
+  let noncompliant = 0
+  let pending = 0
+  let unknown = 0
+  for (const policy of discoveredPolicyItems) {
+    if (policy.disabled || !policy?.compliant) continue
+    switch (policy?.compliant?.toLocaleLowerCase()) {
```

Comment:
Let's just do `toLowerCase` here since this is a value coming from the API which is always English.

Reply:
The same comment applies to severity.